.travis.yml

```diff
@@ -33,7 +33,7 @@ before_script:
 
 script:
   - pip install -e .
-  - py.test -v --cov=./
```

Comment:
I only remove the "-v" from that file, as it will confuse with our command line. It should affect the testing as it only change the content to show. Here "-v" means verbose. ..